### PR TITLE
Add Code Coverage Measurement

### PR DESCRIPTION
**Pull request by blink1073**
When run against the current (incomplete) test suite:

screen shot 2015-08-28 at 8 32 02 am

### CODE REVIEW

File: test/karma.conf.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 module.exports = function (config) {
   config.set({
-    frameworks: ['mocha'],
-    reporters: ['mocha'],
-    files: ['build/bundle.js'],
+    frameworks: ['browserify', 'mocha'],
+    reporters: ['mocha', 'coverage'],
```
Comment (sccolbert) [inline test/karma.conf.js]: what does the 'coverage' reporter do for this version of the tests?

Comment (blink1073) [inline test/karma.conf.js]: Ah, remnant, I'll remove it.

File: package.json [hunk 2]
```diff
@@ -32,9 +34,10 @@
   },
   "scripts": {
     "clean": "rimraf lib && rimraf test/build",
+    "coverage": "karma start test/karma-cov.conf.js",
```
Comment (sccolbert) [inline package.json]: can we make this `test:coverage`?

File: scripts/travis_script.sh [hunk 3]
```diff
@@ -5,4 +5,5 @@ set -e
 npm run clean
 npm run build
 npm test
+npm run coverage
```
Comment (sccolbert) [inline scripts/travis_script.sh]: `npm run test:coverage`

**Comment by blink1073** [comment]
Fixed.

**Comment by sccolbert** [comment]
Thanks!

**Pull request merged**